### PR TITLE
Rebalance difficulty

### PR DESCRIPTION
**Pull request by Brett208**
Finally broke down and adjusted the difficulty. Rational for new selection is contained in the commit messages. Selections based on previous plays. I haven't fully tested the new selection, but believe they are a good start.

### CODE REVIEW

File: Main.cpp [hunk 1]
```diff
@@ -168,6 +155,43 @@ Export int InitProc()
...
+	case DiffEasy: {
+		saveData.diffMultiplier = 13;
+		return; }
+	case DiffNormal: {
+		saveData.diffMultiplier = 10;
+		return; }
+	case DiffHard: {
+		saveData.diffMultiplier = 7;
+		return; }
+	}
```
Comment (DanRStevens) [inline Main.cpp]: The brace formatting looks a bit messy. The indentation looks a bit excessive (might be a GitHub display thing).

----

For an alternative to `switch`, you can also use a small lookup table:
```cpp
const int diffMultipliers[] = {13, 10, 7}; // Easy, Medium, Hard
return diffMultipliers[Player[0].Difficulty()];
```

File: Main.cpp [hunk 2]
```diff
@@ -171,6 +171,25 @@ void SetDifficultyMultiplier()
 	}
 }
 
+void SetStartingColonists()
+{
+	switch (Player[0].Difficulty())
+	{
+	case (DiffEasy): {
```
Comment (DanRStevens) [inline Main.cpp]: The parenthesis around the `case` label seem kind of strange.
```cpp
case DiffEasy: {
```

**Review by DanRStevens — APPROVED** [review]
Looks reasonable. Might be good to send BlackBox a message about the changes.

The original values seemed pretty arbitrary. Would be nice to have a bit more context about the changes. Would be neat to see how many colonists are needed for required structures, or how weapons research time compares with initial attack wave times. Not strictly necessary though. Play testing is probably a better indicator of good values.

File: Main.cpp (hunk 1 above)
Comment (Brett208) [reply Main.cpp]: Good idea.

**Comment by Brett208** [comment]
Sent a message to BlackBox. I'll just sit on this branch for a while in the meantime.

Thanks for commenting on using an array. Not sure why I thought the switch was a better idea...

-Brett

**Review by DanRStevens — APPROVED** [review]
Ahh, I see you've found other uses for the table lookup.

Might be helpful to add a comment about the difficulty order for the arrays. (Easy, Medium, Hard). I tend to have trouble remembering what the order is.

**Pull request merged**